package.json

```diff
@@ -25,7 +25,7 @@
 		"linkify-issues": "^1.3.0",
 		"linkify-urls": "^2.2.0",
 		"onetime": "^2.0.1",
-		"select-dom": "^4.2.0-0",
+		"select-dom": "^4.1.3",
```

Comment:
Temporarily went back to the stable. Going to upgrade this again. Was just trying to get tests running.